<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageBox</class>
 <widget class="QDialog" name="MessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MessageBox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="title" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="labTitle">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>204</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="PushButton" name="btnClose">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/Images/images/close_blue.png</normaloff>:/Images/images/close_blue.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="content" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="hlContent" stretch="0,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="icon">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="vlContent" stretch="0,0">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="text">
            <property name="text">
             <string/>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="hlButton">
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <spacer name="hsLeft">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnYes">
              <property name="text">
               <string notr="true">确定</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnNo">
              <property name="text">
               <string notr="true">否</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnCancel">
              <property name="text">
               <string notr="true">取消</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="hsRight">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>PushButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/QtBaseUIResource.qrc"/>
 </resources>
 <connections/>
</ui>
